### PR TITLE
Make world right handed

### PR DESCRIPTION
**Pull request by krupitskas**

### CODE REVIEW

File: UnicornRender/video/include/unicorn/video/Primitives.hpp [hunk 1]
```diff
@@ -57,6 +57,7 @@ class Primitives
     * Supported formats:
     * - gltf 2.0 (without binary glb)
     * - obj
+    * - fbx (without embedded textures)
```
Comment (MrDmitry) [inline UnicornRender/video/include/unicorn/video/Primitives.hpp:60]: how is this related to the world right-handedness?

File: UnicornRender/video/source/OrthographicCamera.cpp [hunk 2]
```diff
@@ -36,6 +36,7 @@ void OrthographicCamera::UpdateProjection()
     m_cameraProjection = glm::ortho(-xScaled, xScaled,
                                     -yScaled, yScaled,
                                     m_back, m_front);
+    m_cameraProjection[1][1] *= -1; //ToDo: For vulkan rendering.
```
Comment (MrDmitry) [inline UnicornRender/video/source/OrthographicCamera.cpp]: what is this `todo` about?

**Review by MrDmitry — CHANGES_REQUESTED** [review]
Please squash unneeded comments and fix commit names. Commit title should be brief and explain what this change does, not the `Now we have ...` bit

File: UnicornRender/video/include/unicorn/video/Primitives.hpp (hunk 1 above)
Comment (krupitskas) [reply UnicornRender/video/include/unicorn/video/Primitives.hpp:60]: FBX was added for visual transformation tests, because gltf has bug and obj bake all transformation in vertices

File: UnicornRender/video/source/OrthographicCamera.cpp (hunk 2 above)
Comment (krupitskas) [reply UnicornRender/video/source/OrthographicCamera.cpp]: This was moved from shader in here.We inverting perspective in vertical axis to render it correctly, because vulkan has it upside-down

File: UnicornRender/video/include/unicorn/video/Transform.hpp [hunk 3]
```diff
@@ -109,13 +109,13 @@ class Transform
...
     /**
-     * @brief Translates by axes
+     * @brief Translates considering current orientation
      *
      * @attention Don't forget to recalculate @sa UpdateTransformMatrix to commit all changes to the object
      *
      * @param[in] distance per component distance
      */
-    UNICORN_EXPORT void TranslateLocal(glm::vec3 distance);
+    UNICORN_EXPORT void TranslateByBasis(glm::vec3 distance);
```
Comment (ilia-glushchenko) [inline UnicornRender/video/include/unicorn/video/Transform.hpp:118]: I don't really get what this function does and how does it differ from TranslateWorld. Please consider extending documentation.

File: UnicornRender/video/source/OrthographicCamera.cpp (hunk 2 above)
Comment (ilia-glushchenko) [reply UnicornRender/video/source/OrthographicCamera.cpp]: I think that it's not a todo than, but a comment.

**Review by ilia-glushchenko — CHANGES_REQUESTED** [review]

**Review by ilia-glushchenko — APPROVED** [review]

**Review by MrDmitry — APPROVED** [review]

**Comment by dsodintsov** [comment]
approve

**Comment by MrDmitry** [comment]
pullapprove is being stupid and doesn't see my approve

**Comment by t0mgit** [comment]
approve

[Approved with PullApprove](https://pullapprove.com/Godlike/Unicorn/pull-request/122/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Godlike/Unicorn)
